Mathlib/Order/Filter/Defs.lean

```diff
@@ -296,17 +296,14 @@ def comap (m : α → β) (f : Filter β) : Filter α where
   inter_sets := fun ⟨a', ha₁, ha₂⟩ ⟨b', hb₁, hb₂⟩ =>
     ⟨a' ∩ b', inter_mem ha₁ hb₁, inter_subset_inter ha₂ hb₂⟩
 
-/-- Product of filters. This is the filter generated by cartesian products
-of elements of the component filters. -/
-protected def prod (f : Filter α) (g : Filter β) : Filter (α × β) :=
-  f.comap Prod.fst ⊓ g.comap Prod.snd
-
 /-- Coproduct of filters. -/
 protected def coprod (f : Filter α) (g : Filter β) : Filter (α × β) :=
   f.comap Prod.fst ⊔ g.comap Prod.snd
 
+/-- Product of filters. This is the filter generated by cartesian products
+of elements of the component filters. -/
 instance instSProd : SProd (Filter α) (Filter β) (Filter (α × β)) where
-  sprod := Filter.prod
+  sprod f g := f.comap Prod.fst ⊓ g.comap Prod.snd
```

Comment:
```suggestion
  sprod f g := f.comap Prod.fst ⊓ g.comap Prod.snd

/-- Product of filters. This is the filter generated by cartesian products
of elements of the component filters.
Deprecated. Use `f ×ˢ g` instead. -/
@[deprecated (since := "2024-11-29")]
protected def prod (f : Filter α) (g : Filter β) : Filter (α × β) := f ×ˢ g
```